### PR TITLE
Reset double confirm

### PR DESCRIPTION
**Pull request by TyrusLye**
Reset now will now prompt the user to require an additional confirmation before resetting the Address Book, regardless of which reset command is entered.

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ClearCommandParser.java [hunk 1]
```diff
@@ -16,19 +17,18 @@ public class ClearCommandParser {
...
-            throw new ParseException(
-                    String.format(ClearCommand.MESSAGE_USAGE));
+            return new ClearCommand("Empty");
         }
         try {
             String confirmation = ParserUtil.parseSimpleString(args);
             if (!confirmation.equals("confirm")) {
                 throw new ParseException(
-                        String.format(ClearCommand.MESSAGE_USAGE));
+                        String.format(ClearCommand.MESSAGE_USAGE + "."));
```
Comment (butteredyakiimo) [inline src/main/java/seedu/address/logic/parser/ClearCommandParser.java]: i don;t think we are allowed to add this full stop, its a cosmetic change!!

File: src/main/java/seedu/address/logic/parser/ClearCommandParser.java [hunk 2]
```diff
@@ -16,19 +17,18 @@ public class ClearCommandParser {
...
             }
             return new ClearCommand(confirmation);
         } catch (ParseException pe) {
             throw new ParseException(
-                    String.format(ClearCommand.MESSAGE_USAGE), pe);
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, ClearCommand.MESSAGE_USAGE + "."), pe);
```
Comment (butteredyakiimo) [inline src/main/java/seedu/address/logic/parser/ClearCommandParser.java]: same as above

**Review by butteredyakiimo — COMMENTED** [review]
I tested it out, and realised that there are now 2 ways to reset :
1. reset + reset confirm 
2. reset confirm + reset confirm

However, the original idea we had was reset + reset confirm is the only way to reset the addressbook, and in 1.3, the bug we had was that `reset confirm` bypassed this double reset (not intended).

However, if we allow the second case now (ie. x2 reset confirm) to reset the addressbook, i feel like it might be a feature freeze violation.
Is it possible to just revert to the original idea (ie. 1), and remove the second way of resettting?

so the only valid way to reset should be reset + rest confirm. Also, remember to not make any cosmetic changes to the error messages, unless they result in like the user performing the incorrect step/ command. but nice work though!

File: src/main/java/seedu/address/logic/parser/ClearCommandParser.java (hunk 2 above)
Comment (TyrusLye) [reply src/main/java/seedu/address/logic/parser/ClearCommandParser.java]: Yup, I got rid of that

File: src/main/java/seedu/address/logic/parser/ClearCommandParser.java (hunk 1 above)
Comment (TyrusLye) [reply src/main/java/seedu/address/logic/parser/ClearCommandParser.java]: Yep, I got rid of that too

**Review by butteredyakiimo — COMMENTED** [review]
lgtm, i think u may need to add more tests to pull up the codecov: see this report for more!
https://app.codecov.io/gh/AY2324S1-CS2103T-T13-4/tp/pull/143/blob/src/main/java/seedu/address/logic/LogicManager.java

**Review by butteredyakiimo — APPROVED** [review]

**Pull request merged**